### PR TITLE
PAI reminder to submit dna

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Adds a small reminder to the owner, to submit their dna

## Why It's Good For The Game
i always forget

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
add: pai reminder to submit dna
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
Needs testing evidence

File: code/game/objects/items/devices/paicard.dm [hunk 1]
```diff
@@ -140,6 +140,11 @@
 	playsound(loc, 'sound/effects/pai_boot.ogg', 50, 1, -1)
 	audible_message("\The [src] plays a cheerful startup noise!")
 
+	var/turf/T = get_turf(src)
+	for(var/mob/M in get_turf(src))
```
Comment (Zonespace27) [inline code/game/objects/items/devices/paicard.dm:144]: this seems absolutely hacky as a solution

Comment (etherware-novice) [reply code/game/objects/items/devices/paicard.dm:144]: well how else am i supposed to get the mob holding the card

Comment (Zonespace27) [reply code/game/objects/items/devices/paicard.dm:144]: Get the loc of the card, check if it's a mob

File: code/game/objects/items/devices/paicard.dm [hunk 2]
```diff
@@ -140,6 +140,11 @@
 	playsound(loc, 'sound/effects/pai_boot.ogg', 50, 1, -1)
 	audible_message("\The [src] plays a cheerful startup noise!")
 
+	var/turf/T = get_turf(src)
```
Comment (Zonespace27) [inline code/game/objects/items/devices/paicard.dm:143]: T var is unused

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/paicard.dm (hunk 1 above)
Comment (etherware-novice) [reply code/game/objects/items/devices/paicard.dm:144]: whats the code for that

Comment (Zonespace27) [reply code/game/objects/items/devices/paicard.dm:144]: Figure it out, it's not complicated at all

**Pull request closed without merging**